client/state/themes/selectors/get-recommended-themes.js

```diff
@@ -1,6 +1,12 @@
+/**
+ * External dependencies
+ */
+import { camelCase } from 'lodash';
```

Comment:
`NB:` [I know that we're trying to wean ourselves off of lodash](p4TIVU-9Bf-p2). We can probably refactor this as a follow-up in the future, but we could probably add a homegrown version of `camelCase` to the Calypso `js-utils` package and reference that instead.

Reply:
I'd leave this out of the scope of this PR. I think the addition should be handled in a separate PR which would also replace all of the existing usages.

Reply:
Good to think about, but I've gone ahead and removed it anyway just to simplify the codebase. I prefer the aesthetics of `recommendedThemes.blockTemplates` over `recommendedThemes['block-templates']` but machines understand the latter just fine.